examples/usage/strings.ipynb

```diff
@@ -263,8 +263,8 @@
       "print(nengo.builder.operator.Reset(sig, tag=\"tag\"))\n",
       "print(nengo.builder.operator.Copy(sig, sig))\n",
       "print(nengo.builder.operator.Copy(sig, sig, tag=\"tag\"))\n",
-      "print(nengo.builder.operator.SlicedCopy(sig, sig, Ellipsis, slice(0, 1)))\n",
-      "print(nengo.builder.operator.SlicedCopy(sig, sig, Ellipsis, slice(0, 1), tag=\"tag\"))\n",
+      "print(nengo.builder.operator.Copy(sig, sig, [0], slice(0, 1)))\n",
+      "print(nengo.builder.operator.Copy(sig, sig, [0], slice(0, 1), tag=\"tag\"))\n",
```

Comment:
Yeah, exactly. I just wanted to see what it looked like, especially since I changed the print function a bit in this PR.